src/sentry/models/repository.py

```diff
@@ -53,6 +53,43 @@ class Meta:
 
     __repr__ = sane_repr("organization_id", "name", "provider")
 
+    @classmethod
+    def get_by_name_case_insensitive(
```

Comment:
I don't think this method is necessary, I'm pretty sure Django supports case-insensitive searching via
```
repository = Repository.objects.get(
    organization_id=project.organization_id,
    name__iexact=commit_comparison.head_repo_name,
    provider=f"integrations:{commit_comparison.provider}",
)
```